packages/node-core/src/indexer/storeCache/cacheModel.ts

```diff
@@ -337,7 +344,16 @@ export class CachedModel<
           this.model.destroy({where: {id: Object.keys(removeRecords)} as any, transaction: tx}),
       ]);
     }
-
+    this.exporters.forEach((store: Exporter) => {
+      tx.afterCommit(async () => {
+        try {
+          await store.export(records);
+        } catch (e) {
+          throw new Error(`Failed to write to csv, ${e}`);
+        }
+        console.log('exported to csv');
```

Comment:
I think we can drop the catch here. The store should throw an appropriate error.

If an error is throw does it give an appropriate error?